.changeset/cuddly-news-flow.md

```diff
@@ -0,0 +1,11 @@
+---
+"thirdweb": patch
```

Comment:
Question: what makes this a patch instead of minor?

Reply:
should be minor, fixing